src/main/java/org/opensearch/sdk/rest/ExtensionRestPathRegistry.java

```diff
@@ -85,7 +104,7 @@ private void registerHandler(Method method, String path, ExtensionRestHandler ex
     private void registerAsDeprecatedHandler(Method method, String path, ExtensionRestHandler handler, String deprecationMessage) {
         assert (handler instanceof ExtensionDeprecationRestHandler) == false;
 
-        registerHandler(method, path, new ExtensionDeprecationRestHandler(handler, deprecationMessage, deprecationLogger));
+        registerHandler(method, path, null, null, new ExtensionDeprecationRestHandler(handler, deprecationMessage, deprecationLogger));
```

Comment:
Nit: Instead of passing null here we can have a separate method for  `registerHandler` without `name` and `actionNames.`